languages/csharp/exercises/concept/exceptions/.meta/Example.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+static class SimpleCalculator
+{
+    public static string ErrorLog {get; set;}
+    public static int Calculate(int operand1, int operand2, string operation)
```

Comment:
Maybe we could have the student throw an `ArgumentNullException` if `operation` is `null`? It is one of the most common exceptions so it might be useful to have the students work with them here.

Reply:
An extra check could be to throw an `ArgumentException` if the `operation` parameter is empty.